msal/tests/Test.MSAL.NET.Unit/CacheTests/TokenCacheTests.cs

```diff
@@ -276,6 +281,51 @@ public void GetExpiredAccessTokenTest()
             }
         }
 
+        [TestMethod]
+        [TestCategory("TokenCacheTests")]
+        public void GetExpiredAccessToken_WithExtendedExpireStillValid_Test()
+        {
+            using (var httpManager = new MockHttpManager())
+            {
+                httpManager.AddInstanceDiscoveryMockHandler();
+
+                _cache = new TokenCache()
+                {
+                    ClientId = TestConstants.ClientId,
+                    HttpManager = httpManager
+                };
+
+                var atItem = new MsalAccessTokenCacheItem(
+                    TestConstants.ProductionPrefNetworkEnvironment,
+                    TestConstants.ClientId,
+                    "Bearer",
+                    TestConstants.Scope.AsSingleString(),
+                    TestConstants.Utid,
+                    null,
+                    new DateTimeOffset(DateTime.UtcNow),
+                    new DateTimeOffset(DateTime.UtcNow + TimeSpan.FromHours(2)),
+                    MockHelpers.CreateClientInfo());
+
+                atItem.Secret = atItem.GetKey().ToString();
```

Comment:
>atItem.Secret = atItem.GetKey().ToString(); [](start = 16, length = 43)

any reason this is used for the secret?

Reply:
not. a. clue.  was just trying to be consistent with other tests for now.